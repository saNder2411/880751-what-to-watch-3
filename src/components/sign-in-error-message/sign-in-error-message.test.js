import React from 'react';
import renderer from 'react-test-renderer';
import SignInErrorMessage from './ sign-in-error-message.jsx';

const error = {
  status: 400,
};

describe(`Render SignInErrorMessage`, () => {
  it(`Should SignInErrorMessage render correctly when bad validation email`, () => {
    const markup = renderer
      .create(<SignInErrorMessage isValidEmail={false} />)
      .toJSON();

    expect(markup).toMatchSnapshot();
  });

  it(`Should SignInErrorMessage render correctly when bad request data`, () => {
    const markup = renderer
      .create(<SignInErrorMessage error={error} isValidEmail={true} />)
      .toJSON();

    expect(markup).toMatchSnapshot();
  });
});

