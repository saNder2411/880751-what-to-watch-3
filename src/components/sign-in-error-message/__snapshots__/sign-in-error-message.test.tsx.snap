// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render SignInErrorMessage Should SignInErrorMessage render correctly when bad request data 1`] = `
<div
  className="sign-in__message"
>
  <p>
    We canâ€™t recognize this email 
    <br />
     and password combination. Please try again.
  </p>
</div>
`;

exports[`Render SignInErrorMessage Should SignInErrorMessage render correctly when bad validation email 1`] = `
<div
  className="sign-in__message"
>
  <p>
    Please enter a valid email address
  </p>
</div>
`;
